--- config.def.h	2013-04-20 08:29:39.000000000 -0500
+++ config.h	2014-02-07 04:43:29.057988203 -0600
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#201E24", /*color0*/
+    "#2D2B30", /*color1*/
+    "#3F3D40", /*color2*/
+    "#413C3E", /*color3*/
+    "#423E41", /*color4*/
+    "#4E4A4B", /*color5*/
+    "#6E6765", /*color6*/
+    "#837E81", /*color7*/
+
+    /* 8 bright colors */
+    "#837C78", /*color8*/
+    "#89827D", /*color9*/
+    "#A7A099", /*color10*/
+    "#C2BEC1", /*color11*/
+    "#C3BCB2", /*color12*/
+    "#CBC5BA", /*color13*/
+    "#E3DDD2", /*color14*/
+    "#EFE8DC", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#000000", /*background*/
+    "#d3d3d3", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
