--- config.def.h	2013-04-20 21:29:39.000000000 +0800
+++ config.h	2014-02-18 01:18:18.816989068 +0800
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#3e4146", /*color0*/
+    "#9da6a4", /*color1*/
+    "#94a8a8", /*color2*/
+    "#a2a8a3", /*color3*/
+    "#87979c", /*color4*/
+    "#808391", /*color5*/
+    "#799494", /*color6*/
+    "#c3c7bf", /*color7*/
+
+    /* 8 bright colors */
+    "#2a2e35", /*color8*/
+    "#bfc5bf", /*color9*/
+    "#afc7c7", /*color10*/
+    "#bbc2bc", /*color11*/
+    "#a1afb5", /*color12*/
+    "#adb0c4", /*color13*/
+    "#a3c7c7", /*color14*/
+    "#ced1ca", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#2a2e35", /*background*/
+    "#ccc2a6", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
