--- config.def.h	2013-04-20 21:29:39.000000000 +0800
+++ config.h	2014-02-09 04:10:21.799921735 +0800
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#05060F", /*color0*/
+    "#0C1126", /*color1*/
+    "#29272F", /*color2*/
+    "#2C3547", /*color3*/
+    "#464042", /*color4*/
+    "#615D5D", /*color5*/
+    "#7C7F7E", /*color6*/
+    "#8C7566", /*color7*/
+
+    /* 8 bright colors */
+    "#975C45", /*color8*/
+    "#9F8B75", /*color9*/
+    "#A89F93", /*color10*/
+    "#BDC1B9", /*color11*/
+    "#C8A077", /*color12*/
+    "#CBB59E", /*color13*/
+    "#DECCB3", /*color14*/
+    "#F2E6D8", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#111111", /*background*/
+    "#d3d3d3", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
