--- config.def.h	2013-04-20 21:29:39.000000000 +0800
+++ config.h	2014-02-09 04:12:31.124304882 +0800
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#0B0601", /*color0*/
+    "#2C1605", /*color1*/
+    "#3D210B", /*color2*/
+    "#421E05", /*color3*/
+    "#4F2709", /*color4*/
+    "#6E350B", /*color5*/
+    "#7A4217", /*color6*/
+    "#853D0A", /*color7*/
+
+    /* 8 bright colors */
+    "#904711", /*color8*/
+    "#9C6F4A", /*color9*/
+    "#AE5210", /*color10*/
+    "#E26B14", /*color11*/
+    "#FAF7A2", /*color12*/
+    "#FDF24C", /*color13*/
+    "#FE9D1C", /*color14*/
+    "#FEE42A", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#170F0D", /*background*/
+    "#746C48", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
