--- config.def.h	2013-04-20 21:29:39.000000000 +0800
+++ config.h	2014-02-09 04:16:49.634073115 +0800
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#191F31", /*color0*/
+    "#282E39", /*color1*/
+    "#303844", /*color2*/
+    "#3B444A", /*color3*/
+    "#3C4435", /*color4*/
+    "#4E534F", /*color5*/
+    "#5D5D35", /*color6*/
+    "#66695D", /*color7*/
+
+    /* 8 bright colors */
+    "#706D56", /*color8*/
+    "#8B7852", /*color9*/
+    "#909581", /*color10*/
+    "#91703A", /*color11*/
+    "#948658", /*color12*/
+    "#A19662", /*color13*/
+    "#AD9152", /*color14*/
+    "#C6A13F", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#170F0D", /*background*/
+    "#746C48", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
