--- config.def.h	2013-04-20 21:29:39.000000000 +0800
+++ config.h	2014-02-09 04:28:33.892268680 +0800
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#1C1C19", /*color0*/
+    "#2E2C29", /*color1*/
+    "#46413B", /*color2*/
+    "#5A5F55", /*color3*/
+    "#6F9558", /*color4*/
+    "#737987", /*color5*/
+    "#76978F", /*color6*/
+    "#76B64C", /*color7*/
+
+    /* 8 bright colors */
+    "#95A39D", /*color8*/
+    "#98A267", /*color9*/
+    "#A76B5E", /*color10*/
+    "#A7C4BF", /*color11*/
+    "#C9AE6E", /*color12*/
+    "#E3AE98", /*color13*/
+    "#E4CAAE", /*color14*/
+    "#E9D5CC", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#170F0D", /*background*/
+    "#746C48", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
