--- config.def.h	2013-04-20 21:29:39.000000000 +0800
+++ config.h	2014-02-22 10:06:25.677254289 +0800
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#0B0B0B", /*color0*/
+    "#262516", /*color1*/
+    "#303030", /*color2*/
+    "#4C4B31", /*color3*/
+    "#4F4F4F", /*color4*/
+    "#666658", /*color5*/
+    "#6E6E6E", /*color6*/
+    "#7F796A", /*color7*/
+
+    /* 8 bright colors */
+    "#8E8E8E", /*color8*/
+    "#A1A19E", /*color9*/
+    "#ACA05A", /*color10*/
+    "#B0B0B0", /*color11*/
+    "#CECECE", /*color12*/
+    "#D1C9AA", /*color13*/
+    "#E3E0DB", /*color14*/
+    "#E7E7E7", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#111111", /*background*/
+    "#d3d3d3", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
