--- config.def.h	2013-04-20 08:29:39.000000000 -0500
+++ config.h	2014-02-07 02:51:21.884629952 -0600
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#343436", /*color0*/
+    "#473C37", /*color1*/
+    "#6A6E39", /*color2*/
+    "#6D533A", /*color3*/
+    "#8F6F4A", /*color4*/
+    "#A0883A", /*color5*/
+    "#AD8F62", /*color6*/
+    "#CD9561", /*color7*/
+
+    /* 8 bright colors */
+    "#3C4537", /*color8*/
+    "#544639", /*color9*/
+    "#6C9242", /*color10*/
+    "#735E45", /*color11*/
+    "#99663A", /*color12*/
+    "#A97749", /*color13*/
+    "#C47B40", /*color14*/
+    "#CFBDA4", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#170F0D", /*background*/
+    "#746C48", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
