--- config.def.h	2013-04-20 08:29:39.000000000 -0500
+++ config.h	2014-02-07 04:04:06.738274092 -0600
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#FFFFFF", /*color0*/
+    "#121212", /*color1*/
+    "#A5A5A5", /*color2*/
+    "#777777", /*color3*/
+    "#999999", /*color4*/
+    "#4C4C4C", /*color5*/
+    "#2B2B2B", /*color6*/
+    "#000000", /*color7*/
+
+    /* 8 bright colors */
+    "#9A9A9A", /*color8*/
+    "#1C1C1C", /*color9*/
+    "#A4A4A4", /*color10*/
+    "#646464", /*color11*/
+    "#7C7C7C", /*color12*/
+    "#515151", /*color13*/
+    "#322929", /*color14*/
+    "#101010", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#FFFFFF", /*background*/
+    "#000000", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
