--- config.def.h	2013-04-20 08:29:39.000000000 -0500
+++ config.h	2014-02-08 00:36:15.061039750 -0600
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#221B25", /*color0*/
+    "#2C2A35", /*color1*/
+    "#323546", /*color2*/
+    "#453D43", /*color3*/
+    "#547787", /*color4*/
+    "#5D5455", /*color5*/
+    "#6E8F93", /*color6*/
+    "#6F7570", /*color7*/
+
+    /* 8 bright colors */
+    "#79847A", /*color8*/
+    "#8E8979", /*color9*/
+    "#8E6E60", /*color10*/
+    "#8F9485", /*color11*/
+    "#90543F", /*color12*/
+    "#A5A894", /*color13*/
+    "#C69B79", /*color14*/
+    "#D2B897", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#170f0d", /*background*/
+    "#746c48", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
