--- config.def.h	2013-04-20 21:29:39.000000000 +0800
+++ config.h	2014-02-09 03:49:16.099745089 +0800
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#251B27", /*color0*/
+    "#4B3637", /*color1*/
+    "#523F40", /*color2*/
+    "#675455", /*color3*/
+    "#6D5552", /*color4*/
+    "#88573B", /*color5*/
+    "#8C7C83", /*color6*/
+    "#8F746D", /*color7*/
+
+    /* 8 bright colors */
+    "#906A59", /*color8*/
+    "#9C8379", /*color9*/
+    "#A19696", /*color10*/
+    "#A98A77", /*color11*/
+    "#AA958C", /*color12*/
+    "#AEA9A9", /*color13*/
+    "#C49F7A", /*color14*/
+    "#D0CFC1", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#1f1b18", /*background*/
+    "#b9ae9e", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
