--- config.def.h	2013-04-20 21:29:39.000000000 +0800
+++ config.h	2014-02-09 03:53:39.567449221 +0800
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#3B2229", /*color0*/
+    "#405156", /*color1*/
+    "#4D2C30", /*color2*/
+    "#4F3946", /*color3*/
+    "#5F5253", /*color4*/
+    "#608E89", /*color5*/
+    "#643537", /*color6*/
+    "#6D8773", /*color7*/
+
+    /* 8 bright colors */
+    "#9A6054", /*color8*/
+    "#9D8F75", /*color9*/
+    "#9E9E8E", /*color10*/
+    "#B96057", /*color11*/
+    "#B9B9B5", /*color12*/
+    "#D09173", /*color13*/
+    "#D5A88D", /*color14*/
+    "#D7CAB9", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#170f0d", /*background*/
+    "#b9ae9e", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
