--- config.def.h	2013-04-20 21:29:39.000000000 +0800
+++ config.h	2014-02-09 03:59:46.844859739 +0800
@@ -28,30 +28,33 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
+    /* 8 normal colors */
+    "#11130D", /*color0*/
+    "#1D231A", /*color1*/
+    "#2D3424", /*color2*/
+    "#2D1A08", /*color3*/
+    "#302713", /*color4*/
+    "#4E2C10", /*color5*/
+    "#505A38", /*color6*/
+    "#635128", /*color7*/
+
+    /* 8 bright colors */
+    "#673A15", /*color8*/
+    "#7C874F", /*color9*/
+    "#9B6327", /*color10*/
+    "#A1A554", /*color11*/
+    "#BD7C30", /*color12*/
+    "#BFA549", /*color13*/
+    "#CA9339", /*color14*/
+    "#D2AF4A", /*color15*/
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc", /*cursor*/
+    "#170f0d", /*background*/
+    "#746c48", /*foreground*/
+    "#303030",
 };
 
 
@@ -59,8 +62,8 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 
 /*
@@ -329,4 +332,3 @@
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
